broker-daemon/broker-rpc/wallet-service/withdraw-funds.js

```diff
@@ -22,8 +22,12 @@ async function withdrawFunds ({ params, relayer, logger, engines }, { WithdrawFu
     throw new PublicError(`No engine available for ${symbol}`)
   }
 
-  const multiplier = currencies.find(({ symbol: configSymbol }) => configSymbol === symbol).quantumsPerCommon
-  const amountInSat = (Big(amount) * multiplier)
+  const multiplier = (currencies.find(({ symbol: configSymbol }) => configSymbol === symbol) || {}).quantumsPerCommon
```

Comment:
[nit] can be cleaned up to use: `const { quantumsPerCommon: multiplier } = currencies.find(...) || {}`